docs/rst/introduction/introduction.rst

```diff
@@ -0,0 +1,69 @@
+.. _vulcanexus_global_introduction:
+
+Vulcanexus Overview
+===================
+
+*Vulcanexus*, the *ROS 2* all-in-one tool set, expands the *ROS 2* environment improving the developer experience with *ROS 2* providing open source features and tools not available with the current `ROS 2 release <https://www.ros.org/reps/rep-2000.html#release-schedule>`_.
+Consequently, *Vulcanexus* shares the underlying concepts, principles and architecture of :doc:`ROS 2 <../../ros2_documentation/source/index>`.
+For more information about *ROS 2* please refer to the 2022 Science Robotics vol. 7 (May 2022) paper `Robot Operating System 2: Design, architecture, and uses in the wild <https://www.science.org/doi/10.1126/scirobotics.abm6074>`_ by S. Macenski, T. Foote, B. Gerkey, C. Lalancette and W. Woodall.
```

Comment:
I'd include this as a footnote. Check [here](https://devopstutodoc.readthedocs.io/en/stable/documentation/doc_generators/sphinx/rest_sphinx/footnotes.html) how to do so